todocore/src/main/java/com/knighttodo/todocore/character/service/impl/BonusServiceImpl.java

```diff
@@ -2,34 +2,37 @@
 
 import com.knighttodo.todocore.character.domain.BonusVO;
 import com.knighttodo.todocore.character.exception.BonusNotFoundException;
-import com.knighttodo.todocore.character.gateway.BonusGateway;
 import com.knighttodo.todocore.character.service.BonusService;
+import com.knighttodo.todocore.character.service.privatedb.mapper.BonusMapper;
+import com.knighttodo.todocore.character.service.privatedb.repository.BonusRepository;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 @RequiredArgsConstructor
 @Service
 public class BonusServiceImpl implements BonusService {
 
-    private final BonusGateway bonusGateway;
+    private final BonusRepository bonusRepository;
+    private final BonusMapper bonusMapper;
 
     @Override
     public BonusVO save(BonusVO bonusVO) {
-        return bonusGateway.save(bonusVO);
+        return bonusMapper.toBonusVO(bonusRepository.save(bonusMapper.toBonus(bonusVO)));
     }
 
     @Override
     public List<BonusVO> findAll() {
-        return bonusGateway.findAll();
+        return bonusRepository.findAll().stream().map(bonusMapper::toBonusVO).collect(Collectors.toList());
     }
 
     @Override
     public BonusVO findById(String bonusId) {
-        return bonusGateway.findById(bonusId)
-            .orElseThrow(() -> new BonusNotFoundException(
-                String.format("Bonus with such id:%s can't be found", bonusId)));
+        return bonusRepository.findById(bonusId).map(bonusMapper::toBonusVO)
+                .orElseThrow(() -> new BonusNotFoundException(
+                        String.format("Bonus with such id:%s can't be found", bonusId)));
```

Comment:
is it the same formatting that we have in other parts of the project? I suppose that there is a mess but it looks too big of a space before the new line.